File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.kt
class Solution {
    fun solution(num_list: IntArray): Int {
        var num1 = ""
        var num2 = ""
        
        num_list.forEach {
            if (it % 2 == 1) {
                num1 += it.toString()
            } else {
                num2 += it.toString()
            }
        }
        
        return num1.toInt() + num2.toInt()
    }
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    string num1 = "";
    string num2 = "";
    
    for (int n : num_list) {
        if (n % 2) {
            num1 += to_string(n);
        } else {
            num2 += to_string(n);   
        }
    }
    
    return stoi(num1) + stoi(num2);
}